src/db/spanner/manager/session.rs

```diff
@@ -51,6 +50,7 @@ pub async fn create_spanner_session(
         } else {
             // Requires
             // GOOGLE_APPLICATION_CREDENTIALS=/path/to/service-account.json
+            metrics.start_timer("storage.pool.grpc_auth", None);
```

Comment:
I _think_ this belongs here (I moved it in a prior PR)

Reply:
`start_timer` records the time on drop. This metric is in place to monitor how long it takes for the Google Auth to happen (we were seeing a few instances of this taking a long time and want to track to spot anomalies). Keeping the metric within the same block as the credential call should provide the most accurate timing.